Tests/AppStoreConnectClientTests/AppStoreConnectClientTests.swift

```diff
@@ -27,7 +27,7 @@ import XCTest
 
 final class AppStoreConnectClientTests: XCTestCase {
     func testFetchAppsSuccess() async throws {
-        var mockClient = MockAPIClient()
+        var mockClient = MockAPIClient(result: .ok)
```

Comment:
Why we need this change? Looks like the next line (mockClient.result = .ok) does the same

Reply:
Changed.